### PR TITLE
Release 1.0.10

### PR DESCRIPTION
**Pull request by enkore**

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -1,84 +1,95 @@
...
+.. automodule:: borg.repository
     :members:
     :undoc-members:
 
 .. automodule:: borg.hashindex
     :members:
     :undoc-members:
 
-.. automodule:: borg.compress
-    :members: get_compressor, Compressor, CompressorBase
```
Comment (ThomasWaldmann) [inline docs/api.rst]: this was lost.

the release todo checklist is in fact not quite complete here, but we have a bug for the build api being problematic at least.

Comment (enkore) [inline docs/api.rst]: I'm also going to re-remove the non-linux platform modules.

File: setup.py [hunk 2]
```diff
@@ -230,10 +230,13 @@ def run(self):
...
         with open("docs/api.rst", "w") as doc:
             doc.write("""
+.. IMPORTANT: this file is auto-generated by "setup.py build_api", do not edit!
+
+
 API Documentation
 =================
 """)
-            for mod in glob('borg/*.py') + glob('borg/*.pyx'):
+            for mod in sorted(glob('borg/*.py') + glob('borg/*.pyx')):
```
Comment (ThomasWaldmann) [inline setup.py]: ah, cool :)

File: docs/api.rst (hunk 1 above)
Comment (enkore) [inline docs/api.rst]: Hm, checked without the explicit :members:, Sphinx (1.5.2) found them nonetheless.

**Comment by enkore** [comment]
Released

**Pull request merged**